internal/acceptance/registered_model_test.go

```diff
@@ -0,0 +1,43 @@
+package acceptance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+)
+
+func TestAccRegisteredModel(t *testing.T) {
+	name := fmt.Sprintf("terraform-test-registered-model-%[1]s",
+		acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum))
+	workspaceLevel(t,
+		step{
+			Template: fmt.Sprintf(`
+			resource "databricks_registered_model" "model" {
+				name = "%[1]s"
+				catalog_name = "main"
+				schema_name = "default"
+			}
+			
+			resource "databricks_grants" "model_grants" {
+				model = databricks_registered_model.model.id
+			  
+				grant {
+				  principal = "account users"
+				  privileges = ["EXECUTE"]
+				}
+			
+		`, name),
+		},
+		step{
+			Template: fmt.Sprintf(`
+			resource "databricks_registered_model" "model" {
+				name = "%[1]s"
+				catalog_name = "main"
+				schema_name = "default"
+				comment = "new comment"
```

Comment:
Can you add another test here which updates the `name`. This would be good because name is part of the full_name that is used in the request https://docs.databricks.com/api/workspace/registeredmodels/update.

Reply:
Added the test